apiary-metastore-listener/src/test/java/com/expedia/apiary/extensions/metastore/listener/ApiarySnsListenerTest.java

```diff
@@ -68,23 +70,35 @@
   @Captor
   private ArgumentCaptor<PublishRequest> requestCaptor;
 
+  private final Table table = new Table();
   private static final String TABLE_NAME = "some_table";
   private static final String DB_NAME = "some_db";
+  private static final List<String> PARTITION_VALUES = ImmutableList.of("value_1", "value_2", "value_3");
+
+  private List<FieldSchema> partitionKeys;
   private ApiarySnsListener snsListener;
 
   @Before
   public void setup() {
     snsListener = new ApiarySnsListener(configuration, snsClient);
     when(snsClient.publish(any(PublishRequest.class))).thenReturn(publishResult);
+
+    FieldSchema partitionColumn1 = new FieldSchema("Column_1", "String", "");
+    FieldSchema partitionColumn2 = new FieldSchema("Column_2", "String", "");
```

Comment:
Make one of these a type that's not `String`

Reply:
Done.